intents/data/student_score2.csv

```diff
@@ -0,0 +1,17 @@
+class,student_name,subject,marks
+7th,A,maths,90
+7th,B,maths,70
+7th,A,Social science,90
+7th,B,Social science,70
+7th,A,science,90
+7th,B,science,70
+7th,A,english,30
+7th,B,english,90
+8th,B,maths,90
+8th,A,maths,90
+8th,B,Social science,90
+8th,A,Social science,90
+8th,B,science,90
+8th,A,science,90
+8th,B,english,90
+8th,A,english,90
```

Comment:
add end of line

Reply:
I am not sure how to do this for data file

Reply:
Simply go to that line & press enter.

Reply:
done